Gui/opensim/tracking/src/org/opensim/tracking/InverseDynamicsToolPanel.form

```diff
@@ -252,7 +252,7 @@
               <SubComponents>
                 <Component class="javax.swing.JLabel" name="jLabel11">
                   <Properties>
-                    <Property name="text" type="java.lang.String" value="Directory"/>
+                    <Property name="text" type="java.lang.String" value="Storage file"/>
```

Comment:
I would not call this "Storage file" as it is generic name, I'd go with "Forces file" or "Generalized forces file" if space allows. Otherwise looks good. 
Any thoughts or feedback @jimmyDunne or @chrisdembia

Reply:
I like “generalized forces file”